public/app/features/dashboard/components/SaveDashboard/SaveDashboardDrawer.tsx

```diff
@@ -0,0 +1,175 @@
+
+  const data = useMemo<SaveDashboardData>(() => {
+    const clone = dashboard.getSaveModelClone({
+      saveTimerange: Boolean(saveTimerange),
+      saveVariables: Boolean(saveVariables),
+    });
+
+    if (!previous.value) {
+      return { clone, diff: {}, diffCount: 0, hasChanges: false };
+    }
+
+    const cloneJSON = JSON.stringify(clone, null, 2);
+    const cloneSafe = JSON.parse(cloneJSON); // avoids undefined issues
+
+    const diff = jsonDiff(previous.value, cloneSafe);
+    let diffCount = 0;
+    for (const d of Object.values(diff)) {
+      diffCount += d.length;
+    }
+
+    return {
+      clone,
+      diff,
+      diffCount,
+      hasChanges: diffCount > 0 && !status.isNew,
+    };
+  }, [dashboard, previous.value, saveTimerange, saveVariables, status.isNew]);
+
+  const [showDiff, setShowDiff] = useState(false);
+  const { state, onDashboardSave } = useDashboardSave(dashboard);
+  const onSuccess = onSaveSuccess
+    ? () => {
+        onDismiss();
+        onSaveSuccess();
+      }
+    : onDismiss;
+
+  const renderBody = () => {
+    if (showDiff) {
+      return <SaveDashboardDiff diff={data.diff} oldValue={previous.value} newValue={data.clone} />;
+    }
+
+    if (status.isNew || isCopy) {
+      return (
+        <SaveDashboardAsForm
+          dashboard={dashboard}
+          onCancel={onDismiss}
+          onSuccess={onSuccess}
+          onSubmit={onDashboardSave}
+          isNew={status.isNew}
+        />
+      );
+    }
+
+    if (status.isProvisioned) {
+      return <SaveProvisionedDashboardForm dashboard={dashboard} onCancel={onDismiss} onSuccess={onSuccess} />;
+    }
+
+    return (
+      <SaveDashboardForm2
+        dashboard={dashboard}
+        saveModel={data}
+        onCancel={onDismiss}
+        onSuccess={onSuccess}
+        onSubmit={onDashboardSave}
+        options={{
+          saveTimerange,
+          saveVariables,
+        }}
+      />
+    );
+  };
+
+  if (state.error) {
+    return (
+      <SaveDashboardErrorProxy
+        error={state.error}
+        dashboard={dashboard}
+        dashboardSaveModel={data.clone}
+        onDismiss={onDismiss}
+      />
+    );
+  }
+
+  return (
+    <Drawer
+      title={dashboard.title}
+      onClose={onDismiss}
+      width={'40%'}
+      subtitle={
+        <>
+          {hasTimeChanged && (
+            <Checkbox
+              checked={saveTimerange}
+              onChange={() => setSaveTimerange(!saveTimerange)}
+              label="Save current time range as dashboard default"
+              aria-label={selectors.pages.SaveDashboardModal.saveTimerange}
+            />
+          )}
+          {hasVariableChanged && (
+            <Checkbox
+              checked={saveVariables}
+              onChange={() => setSaveVariables(!saveVariables)}
+              label="Save current variable values as dashboard default"
+              aria-label={selectors.pages.SaveDashboardModal.saveVariables}
+            />
+          )}
```

Comment:
Having these options outside the tab feels awkward, I know these impact the Changes tab. But so should name & folder options (in the Save as version). So feels strange to have these options outside the Save tab (Which I would like to rename to Options).